B827EBFFFE0D13B4.json

```diff
@@ -0,0 +1,19 @@
+
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE0D13B4",
+    "servers": [
+      {
+        "server_address": "router.us-west.thethings.network",
```

Comment:
US-West is not a valid server address, check here for the existing ones: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses